common/src/main/java/org/apache/falcon/entity/CatalogStorage.java

```diff
@@ -90,6 +90,7 @@ protected CatalogStorage(Feed feed) throws URISyntaxException {
 
     public CatalogStorage(Cluster cluster, CatalogTable table) throws URISyntaxException {
         this(ClusterHelper.getInterface(cluster, Interfacetype.REGISTRY).getEndpoint(), table);
+        verifyAndUpdateConfiguration(getConf());
```

Comment:
You don't need it here. verifyAndUpdateConfiguration is already called in CatalogStorage(String catalogUrl, CatalogTable table).